src/RecordingStreamer.cpp

```diff
@@ -100,18 +100,26 @@ long long RecordingStreamer::SeekRecordedStream(long long iPosition, int iWhence
   return cur_pos_;
 }
 
-long long RecordingStreamer::PositionRecordedStream(void)
+long long RecordingStreamer::LengthRecordedStream(void)
 {
-  return cur_pos_;
+  return recording_size_;
 }
 
-long long RecordingStreamer::LengthRecordedStream(void)
+PVR_ERROR RecordingStreamer::GetStreamTimes(PVR_STREAM_TIMES* stream_times)
 {
-  return recording_size_;
+  if (stream_times != NULL)
+  {
+    stream_times->startTime = 0;
+    stream_times->ptsStart = 0;
+    stream_times->ptsBegin = 0;
```

Comment:
This implementation implies that your add-on does not support time shifting. Is this correct? If not, you must supply a real implementation.

Reply:
Nvm, just realised that this implementation is for recordings only